[WARNING] 2025-10-30 11:16:03,731 log Not Found: /favicon.ico
[WARNING] 2025-10-30 11:16:03,731 basehttp "GET /favicon.ico HTTP/1.1" 404 3889
[ERROR] 2025-10-30 11:17:09,321 log Internal Server Error: /about/
Traceback (most recent call last):
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/views.py", line 41, in sentry_patched_render
    return old_render(self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/utils.py", line 1851, in runner
    return sentry_patched_function(*args, **kwargs)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/templates.py", line 77, in rendered_content
    return real_rendered_content.fget(self)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/library.py", line 130, in compile_func
    args, kwargs = parse_bits(
                   ~~~~~~~~~~^
        parser,
        ^^^^^^^
    ...<8 lines>...
        function_name,
        ^^^^^^^^^^^^^^
    )
    ^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/library.py", line 417, in parse_bits
    kwarg = token_kwargs([bit], parser)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 1127, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
    ...<2 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '"this.src='{%' from '"this.src='{%'
[ERROR] 2025-10-30 11:17:09,323 basehttp "GET /about/ HTTP/1.1" 500 280751
[ERROR] 2025-10-30 11:18:49,248 log Internal Server Error: /about/
Traceback (most recent call last):
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/views.py", line 41, in sentry_patched_render
    return old_render(self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/utils.py", line 1851, in runner
    return sentry_patched_function(*args, **kwargs)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/templates.py", line 77, in rendered_content
    return real_rendered_content.fget(self)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 1127, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
    ...<2 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('images/about.png')' from 'static('images/about.png')'
[ERROR] 2025-10-30 11:18:49,252 basehttp "GET /about/ HTTP/1.1" 500 271750
[WARNING] 2025-10-30 11:26:26,059 log Not Found: /favicon.ico
[WARNING] 2025-10-30 11:26:26,060 basehttp "GET /favicon.ico HTTP/1.1" 404 3889
[ERROR] 2025-10-30 11:26:36,367 log Internal Server Error: /about/
Traceback (most recent call last):
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/views.py", line 41, in sentry_patched_render
    return old_render(self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/utils.py", line 1851, in runner
    return sentry_patched_function(*args, **kwargs)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/templates.py", line 77, in rendered_content
    return real_rendered_content.fget(self)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 1127, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
    ...<2 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('images/about.png')' from 'static('images/about.png')'
[ERROR] 2025-10-30 11:26:36,370 basehttp "GET /about/ HTTP/1.1" 500 271730
[ERROR] 2025-10-30 11:26:51,860 log Internal Server Error: /about/
Traceback (most recent call last):
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/views.py", line 41, in sentry_patched_render
    return old_render(self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/utils.py", line 1851, in runner
    return sentry_patched_function(*args, **kwargs)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/templates.py", line 77, in rendered_content
    return real_rendered_content.fget(self)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 1127, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
    ...<2 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('images/about.png')' from 'static('images/about.png')'
[ERROR] 2025-10-30 11:26:51,862 basehttp "GET /about/ HTTP/1.1" 500 271730
[ERROR] 2025-10-30 11:29:42,714 log Internal Server Error: /about/
Traceback (most recent call last):
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/views.py", line 41, in sentry_patched_render
    return old_render(self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/utils.py", line 1851, in runner
    return sentry_patched_function(*args, **kwargs)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/sentry_sdk/integrations/django/templates.py", line 77, in rendered_content
    return real_rendered_content.fget(self)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader.py", line 42, in select_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 518, in parse
    raise self.error(token, e)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/defaulttags.py", line 1531, in do_with
    extra_context = token_kwargs(remaining_bits, parser, support_legacy=True)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 1127, in token_kwargs
    kwargs[key] = parser.compile_filter(value)
                  ~~~~~~~~~~~~~~~~~~~~~^^^^^^^
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 609, in compile_filter
    return FilterExpression(token, self)
  File "/Users/dim-gggl/~/Dev Tools/portfolio_2/.venv/lib/python3.14/site-packages/django/template/base.py", line 710, in __init__
    raise TemplateSyntaxError(
    ...<2 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('images/Dimitri_Gaggioli.png')' from 'static('images/Dimitri_Gaggioli.png')'
[ERROR] 2025-10-30 11:29:42,717 basehttp "GET /about/ HTTP/1.1" 500 272125
[WARNING] 2025-10-30 12:50:03,914 log Not Found: /media/cv/CV_DEVPYTH.pdf
[WARNING] 2025-10-30 12:50:03,915 basehttp "GET /media/cv/CV_DEVPYTH.pdf HTTP/1.1" 404 4152
